### ISSUE TITLE
New FSI-in-Terminal: Alt+Enter doesn't work properly on lines containing comments

### ISSUE THREAD

**Issue by rmunn** [ionide/ionide-vscode-fsharp#167]
VS Code 1.5.3, Ionide-fsharp 2.5.2, Ubuntu Linux 14.04.

If my code has comments at the end of the line, Alt+Enter doesn't send them to FSI properly, because the `;;` is appended at the end of the line rather than on a new line. If I have the following lines:

```fsharp
let a = 1
let b = 2  // Two
```

And I press Alt+Enter on each line in turn, the following gets sent to F# Interactive:

```
let a = 1;;
let b = 2  // Two;;
```

The first one executes right away. The second one doesn't execute, because the `;;` is seen as part of the comment, and needs a newline before the `;;` if it's going to run properly.

Two possible solutions. One would be to just put a newline before every `;;`, so that we actually append `\n;;` to every line sent to FSI. That would be simple, but could be slightly uglier (I kind of like that `let a = 1;;` does not have a newline in my FSI terminal). The other solution would be to parse the last line of what is sent, and find if there is a comment marker before the end of the line. If so, then add a newline before the `;;`. That would be more complicated, but would make the FSI output slightly prettier.

**Comment by Krzysztof-Cieslak** [comment]
I think in previous version FSI integration always done `\n;;`.

**Comment by Krzysztof-Cieslak** [comment]
Released fix (adding `\n` before `;;`) in `2.5.4`. I consider it fixed, but would welcome PR handling this situation in nicer way.
